edycja2/test_rozwiazania.py

```diff
@@ -1,7 +1,17 @@
 from argparse import ArgumentParser
 from collections import namedtuple
 from glob import glob
+import hashlib
 from os.path import getsize, isfile
+import os
+import os.path
+import sys
+
+
+from sqlalchemy import Table, Column, Integer, String, \
```

Comment:
IMO w testerach/generatorach testów nie ma co się trzymać limitu 80 znaków, w przypadkach takich jak ten zmniejsza to czytelność

dwa fragmenty z zen of python:
`Readability counts.`;
`practicality beats purity.` :)